src/Microsoft.Extensions.Logging.Abstractions/IMetricLogger.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Extensions.Logging
+{
+    public interface IMetricLogger
```

Comment:
😍

Reply:
I would add something like `DefineMetric` that returns `Func<double>` this might improve speed of some implementations.